core/src/main/java/de/muenchen/allg/itd51/wollmux/former/BroadcastObjectSelection.java

```diff
@@ -74,7 +74,7 @@ public int getState()
   }
 
   /**
-   * true =&gt; Selektion delete complete first bevor selecting/deselecting the object.
+   * true =&gt; Selektion delete fully bevor selecting/deselecting the object.
```

Comment:
Hmm, I guess Selektion needs translation too, as well as bevor (and swapping verb and object is more idiomatic English, i.e. 'Delete selection fully, before ...)